README.md

```diff
@@ -31,31 +31,36 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. What is the DOM?
+    DOM is the form of Javascript that translates to HTML, as Dan said DOM is the living breathing version of HTML.
 
 2. What is an event?
+    An event is a (an?) User triggered move that starts a code, whether that be a hover, a click, a scroll and many more. 
 
 3. What is an event listener?
+    An event listener is like a function that does exactly what its name states, it waits until the user triggers the event and then runs that function.
```

Comment:
An event listener in JavaScript is a function that waits, or listens, for an event to occur. It triggers the event handler, which is the code that runs once the event is "heard."